meta-mender-qemu/Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN apk update && apk upgrade && \
 ARG VEXPRESS_IMAGE=scripts/docker/empty-file
 ARG UBOOT_ELF=scripts/docker/empty-file
 
-ADD $UBOOT_ELF ./u-boot.elf
-ADD $VEXPRESS_IMAGE ./core-image-full-cmdline-vexpress-qemu.sdimg
+COPY $UBOOT_ELF ./u-boot.elf
+COPY $VEXPRESS_IMAGE .
```

Comment:
I think this includes the entire directory, doesn't it? Maybe `core-image-full-cmdline-vexpress-qemu.*` would be preferable? If that's valid syntax, that is.